temporalio/activity.py

```diff
@@ -135,6 +138,41 @@ def _logger_details(self) -> Mapping[str, Any]:
 _current_context: contextvars.ContextVar[_Context] = contextvars.ContextVar("activity")
 
 
+@dataclass
+class _ActivityCancellationDetailsHolder:
+    _details: Optional[ActivityCancellationDetails] = None
+
+    def set_details(self, details: ActivityCancellationDetails) -> None:
+        self._details = details
+
+    @property
+    def details(self) -> Optional[ActivityCancellationDetails]:
+        return self._details
+
+
+@dataclass(frozen=True)
+class ActivityCancellationDetails:
+    """Provides the reasons for the activity's cancellation. Cancellation details are set once and do not change once set."""
+
+    not_found: bool = False
+    cancelled_requested: bool = False
```

Comment:
```suggestion
    cancel_requested: bool = False
```

I think this is clearer than two past-tense words here

Reply:
yup 🤦 fixed